docs/api-v1.json

```diff
@@ -0,0 +1,1136 @@
+{
+  "openapi": "3.0.3",
+  "permissions_preset": "<br><h3>Permitted Roles</h3><ul><li>ANONYMOUS</li><li>LANDLORD</li><li>STUDENT</li><li>TEAM</li><li>ADMIN</li></ul>",
+  "info": {
+    "title": "CozyCave.ch API Documentation",
+    "description": "This is the REST API documentation for the backend of cozycave.ch.<br><br>For all endpoints: if something is required, the you have to send it on POST, if something is explicitly non-required you don't have to set it at all but if somewhere the required require statement is not set, it may be required to e.g. publish the entity but not necessary for POST. Please see description then. <br>Sending a PUT endpoint usually doesn't require any property, if otherwise, it's marked on the description of the object. <br><br>(auto set) := set by the server automatically even when not send.",
+    "version": "1.2.0",
+    "version-explanation": "APIVersion.MayorVersion.MinorVersion"
+  },
+  "servers": [
+    {
+      "url": "https://sopra-fs22-group-15-server.herokuapp.com/"
+    }
+  ],
+  "security": [
+    {
+      "bearerAuth": []
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "bearerAuth": {
+        "type": "http",
+        "description": "JWT Authorization header using the Bearer scheme. Example: \"Authorization: Bearer {token}\"",
+        "scheme": "bearer",
+        "bearerFormat": "JWT"
+      }
+    },
+    "schemas": {
+      "Error": {
+        "type": "object",
+        "readOnly": true,
+        "properties": {
+          "error": {
+            "type": "string",
+            "description": "status code as text",
+            "example": "I'm a teapot"
+          },
+          "message": {
+            "type": "string",
+            "description": "error message",
+            "example": "used a teapot instead of a coffeepot"
+          },
+          "path": {
+            "type": "string",
+            "description": "path of the request that caused the error",
+            "example": "/v1/coffee/"
```

Comment:
probably needs to be changed.

Reply:
why do you think so? its just an example of how an error looks. not a working example